cmd/lotus-miner/sectors.go

```diff
@@ -1628,12 +1628,31 @@ var sectorsExpiredCmd = &cli.Command{
 				}
 
 				toCheck, err = bitfield.SubtractBitField(toCheck, live)
```

Comment:
super nit: if we change the generic name `toCheck` to something more descriptive i.e. `deadSectors` it will make this slightly more readable.